src/kOS/Module/kOSVesselModule.cs

```diff
@@ -79,7 +79,7 @@ protected override void OnStart()
         public override void OnLoadVessel()
         {
             base.OnLoadVessel();
-            SafeHouse.Logger.SuperVerbose(string.Format("kOSVesselModule OnLoadVessel()!  On {0} ({1})", Vessel.vesselName, ID));
+            SafeHouse.Logger.SuperVerbose(string.Format("kOSVesselModule OnLoadVessel()!  On {0}({1}), {2}({3})", Vessel.vesselName, ID, vessel.vesselName, vessel.id));
```

Comment:
This was just to verify that Vessel (capitalized) and vessel (lowercase) are the same thing.  They're both inherited from KSP's VesselModule class and I wasn't sure if they might differ.  I figured it didn't hurt to leave this information in there so I didn't revert this edit.